doc/modules/model_evaluation.rst

```diff
@@ -2403,34 +2407,92 @@ the difference in errors decreases. Finally, by setting, ``power=2``::
 we would get identical errors. The deviance when ``power=2`` is thus only
 sensitive to relative errors.
 
-.. _d2_tweedie_score:
+.. _d2_score:
 
-D² score, the coefficient of determination
--------------------------------------------
+D² score
+--------
 
-The :func:`d2_tweedie_score` function computes the percentage of deviance
-explained. It is a generalization of R², where the squared error is replaced by
-the Tweedie deviance. D², also known as McFadden's likelihood ratio index, is
-calculated as
+The D² score computes the fraction of deviance explained. 
+It is a generalization of R², where the squared error is generalized and replaced 
+by a deviance of choice :math:`\text{D}(y, \hat{y})`
```

Comment:
Maybe, we should avoid confusion with D2 and name deviance with a small `\text{dev}` instead of `\text{D}`.

Reply:
Good idea, thanks! updated.